src/oscar/apps/dashboard/nav.py

```diff
@@ -86,7 +86,7 @@ def default_access_fn(user, url_name, url_args=None, url_kwargs=None):
     # label that can be loaded by get_class (e.g.
     # 'dashboard.catalogue.app), which then essentially checks
     # INSTALLED_APPS for the right module to load
-    match = re.search('(dashboard[\w\.]*)\.views$', view_module)
+    match = re.search('(dashboard[\w\.]*)\.\w+$', view_module)
```

Comment:
I also already run into this restriction from Oscar before, so I support the plan, to remove it :-)
Reading Fabis explanation and Markus thoughts, it seems good an secure for me.

Tip from Tim: after we have made this change, lets make a new ticket, to remove all the workarounds that has been created due to this problem. 
Mine is here: dashboard.vouchers.views.py.

Reply:
@klausbreuer @Azd325 Good idea, created a ticket for that: https://machtfit.atlassian.net/browse/MAC-1896

Reply:
Direkt mit machen würde ich sagen.

Reply:
et voilà: https://github.com/machtfit/machtfit-oscar/pull/915